packages/help-center/src/components/help-center-odie.tsx

```diff
@@ -122,6 +122,10 @@ export function HelpCenterOdie( {
 		[]
 	);
 
+	const isTestUser = useMemo( () => {
```

Comment:
```suggestion
	const isTestUser = navigator?.userAgent?.includes( 'wp-e2e-tests' );
```

Reply:
That's a great suggestion, thank you

Reply:
I ended up using https://github.com/Automattic/wp-calypso/blob/trunk/client/lib/e2e/index.ts function